app/views/application/_navbar.html.erb

```diff
@@ -0,0 +1,6 @@
+<nav>
+	<h3><%= link_to "Home", root_path %> |
+	<%= link_to "Sign up", new_user_path %> |
+	<%= link_to "Sign out", "/sessions/destroy", :method => :delete %> |
+	<%= link_to "Sign in", login_path %></h3>
```

Comment:
1. Fix indentation.
2. The link texts should be a translation.
3. A suggestion to use an `ul` rather than `h3` and plain links
4. Lastly, a suggestion to use a rails link helper for "/sessions/destroy"

Reply:
1, 2 & 4 done in sha: 6639a31
I will be styling this later and will change the layout